scripts/crawl_zenodo.py

```diff
@@ -241,17 +242,21 @@ def get_zenodo_dois(stored_tokens, passed_tokens, verbose=False):
             "types": [],
             "version": metadata["version"] if "version" in metadata.keys() else None,
             "licenses": [metadata["license"] if "license" in metadata.keys() else {}],
-            "keywords": metadata["keywords"]if "keywords" in metadata.keys() else [],
-            "distributions": {
-                "format": files[0]["type"] if len(files) > 0 and "type" in files[0].keys() else None,
-                "size": files[0]["size"] if len(files) > 0 and "size" in files[0].keys() else None,
-                "unit": {"value": "B"},
-                "access": dataset["links"]["html"]
-            },
+            "keywords": metadata["keywords"] if "keywords" in metadata.keys() else [],
+            "distributions": [
+                {
+                    "formats": list(set(map(lambda x: x["type"], files))) if len(files) > 0 else None,
```

Comment:
@emmetaobrien or @zxenia, do you confirm that `formats` can be a list in the DATS model?

Reply:
@glatard it is a list, yes!